source/fundamentals/transactions.txt

```diff
@@ -160,89 +240,95 @@ order details. Suppose you organize the data updates as follows:
 
    * - ``orders``
      - insert
-     - Record the purchase information
+     - Insert a document that describes the order
 
    * - ``customers``
-     - update
-     - Append the order id to associate it with the customer
+     - update or upsert
+     - Append the ``_id`` from the order document to the order history
+       in the customer document
 
    * - ``inventory``
      - update
-     - Subtract quantity of ordered items
+     - Update the quantities of items available after a purchase
 
-A purchase can fail several ways such as if there's insufficient quantity of
-the item in inventory, if the order couldn't be completed, or if your
-payment system is offline.
+A purchase can fail for the following reasons:
 
-If the payment fails, you can use a transaction to make sure that you
-avoid exposing any partial updates that might cause data consistency issues
-for other operations that depend on that data.
+- Insufficient inventory for an item
+- An order could not be completed
+- The system went offline
+
+If the payment fails, you can perform a transaction that includes all of
+the data changes to ensure you avoid making any partial updates that
+might cause data consistency issues for future operations.
 
 Sample Data
 ~~~~~~~~~~~
 
-The code examples require the following sample data in the ``testdb``
-database to run the multi-document payment transaction:
+The code examples use the following sample data in the ``testdb``
```

Comment:
Suggestion:

I couldn't find more than one example (Core API Implementation).
Could it make sense to entitle the parent header "Core API Example" and then move the admonition about the Convenient Transaction API usage example directly after that title?

Reply:
not relevant as I moved the example to its own page